aspnetcore/blazor/debug.md

```diff
@@ -45,54 +44,117 @@ Debugging requires either of the following browsers:
 * Google Chrome (version 70 or later)
 * Microsoft Edge preview ([Edge Dev Channel](https://www.microsoftedgeinsider.com))
 
+## Enable debugging
+
+To enable debugging in an existing Blazor WebAssembly app, update *launchSettings.json* in the startup project of your app. Set `useWebAssemlbyDebugging` to `true` and include the following `inspectUri` property in each launch profile:
+ 
+```json
+"inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}"
+```
+ 
+Once updated, the *launchSettings.json* file should look something like the following:
+ 
+```json
+{
+  "useWebAssemblyDebugging": true,
```

Comment:
The `useWebAssemblyDebugging` property actually isn't needed. It still shows up in our templates, but we'll get that cleaned up in the next release.

```suggestion
```